src/units/race.hpp

```diff
@@ -31,12 +31,12 @@ class unit_race
 	const config& get_cfg() const { return cfg_; }
 	const std::string& id() const { return id_; }
 	const std::string& editor_icon() const { return icon_; }
-	const t_string& name(GENDER gender=MALE) const { return name_[gender]; }
+	const t_string& name(const unit_gender* gender = nullptr) const;
```

Comment:
I think many functions should stop providing default values for their arguments. For example, by removing the default here we find that the single function which depends on it is the implementation of the Lua API; thus there's a Lua API which is assuming the gender.